vere/raft.c

```diff
@@ -1868,7 +1869,14 @@ _raft_grab(u3_noun rus)
       c3_c* wen_c = u3r_string(u3A->wen);
 
       c3_c nam_c[2048];
-      snprintf(nam_c, 2048, "%s/.urb/put/%s-mass.txt", u3_Host.dir_c, wen_c);
+      snprintf(nam_c, 2048, "%s/.urb/put/mass", u3_Host.dir_c);
+
+      struct stat st;
+      if ( -1 == stat(nam_c, &st) ) {
+        mkdir(nam_c, 0700);
+      }
+
+      snprintf(nam_c, 2048, "%s/%s.txt", nam_c, wen_c);
```

Comment:
I think this nested directly structure is fine. Do you think it's worth putting the ship name in the file for ease of later handling?

Reply:
sure that sounds good.